roofit/multiprocess/res/RooFit/MultiProcess/ProcessTimer.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   ZW, Zef Wolffs, Nikhef, zefwolffs@gmail.com
+ *   PB, Patrick Bos, Netherlands eScience Center, p.bos@esciencecenter.nl
+ *
+ * Copyright (c) 2022, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#ifndef ROOT_ROOFIT_MultiProcess_ProcessTimer
+#define ROOT_ROOFIT_MultiProcess_ProcessTimer
+
+#include <chrono>
+#include <string>
+#include <sys/types.h> // pid_t
+#include <map>
+#include <list>
+#include <nlohmann/json.hpp>
+using json = nlohmann::json;
+
+using namespace std;
```

Comment:
```suggestion
```
Remember the C++ rule to never do `using namespace` in header files. Everybody who includes the header will get these using statements too, and things could become ambiguous.